docs/index.md

```diff
@@ -174,6 +174,16 @@ Git had conquered the world of code because it had best supported engineering be
 
 lakeFS provides these exact benefits, that are data practitioners are missing today, and enables them a clear intuitive git-like inetrface to easily manage their data like they manage code. Therefore, lakeFS can definitely be regarded as git for data.
 
+### How is lakeFS different from Git?
+
+An obvious question might be: Why not actually use “Git”, for data (...as opposed to a solution that provides “Git for data” :) )
+
+A basic requirement of working with big data, is that the data stays in place. When writing code, it is common to “check out” a local version of the code to your desktop and work against a copy of the code. However, when working with data, it is unreasonable to copy terabytes (or more) of files locally to experiment, develop, test or transform the data.
```

Comment:
It feels like this para and the next conflate concern with where the data is (local vs object store) with whether a copy is made of it or not (regardless of where that copy it). 
Since we're trying to answer "how is lakeFS different from Git" maybe we need to address (a) why not just use git [git makes local copies, which we don't want to do because it's impractical & inefficient], and then (b) at a simple level how does lakeFS differ in its approach?